sdk/deviceupdate/azure-iot-deviceupdate/src/main/java/com/azure/iot/deviceupdate/implementation/PollingUtils.java

```diff
@@ -0,0 +1,108 @@
+// Copyright (c) Microsoft Corporation. All rights reserved.
```

Comment:
actually most of this class is likely not needed, since you only need to handle `BinaryData` (i.e., U and T in poller is just `BinaryData`).

but fine to keep the logic simple and keep the poller class unchanged.

Reply:
but if you need to suppress in spotbugs, personally I prefer just delete them.

Reply:
All this code is autogenerated. I am not going to touch it. (if it was question for me).